api/v1alpha1/drpolicy_types.go

```diff
@@ -20,11 +20,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type Region string
+
 // Managed cluster information
 type ManagedCluster struct {
 	// Name of this managed cluster as configured in OCM/ACM
 	Name string `json:"name"`
 
+	// Region of a managed cluster determines it DR group.
+	// All managed clusters in a region are considered to be in a sync group.
+	Region Region `json:"region"`
```

Comment:
(question) For Metro case, regions have to be equal and an additional zone would be specified?

Reply:
No, we figured that the zone attribute is not required. All clusters that belong to the same region are part of the sync DR group.